woocommerce/wc-admin
# Update/6487 target audience of business feature step

Fixes #6487 & #6514

Removes the old business bundle code, and removes some of the restriction for showing the new business feature tab with the dropdown.
Also adding a feature flag, incase we want to turn this off, this defaults too true.

### Accessibility




### Screenshots

<img width="987" alt="Screen Shot 2021-03-04 at 2 25 43 PM" src="https://user-images.githubusercontent.com/2240960/110011293-a14de300-7cf5-11eb-9241-08112236f9cf.png">

### Detailed test instructions:
Scenario # 1:
- Start a new store and navigate to the onboarding wizard **WooCommerce > Home**
- Select one of the countries in this list: AU, CA, GB, IE, PR and NZ (US was already supported)
- Select any one of the industries aside from the CBD option.
- Go the the business details tab, it should not be showing the checkbox at the bottom with the free features bundle
- Fill out the dropdown's and click continue
- It should show the `Included business features` tab, allowing you to expand the list, and unselect individual features
- Make sure that `WooCommerce Payments` is part of this list (keep it selected).
- Finish the onboarding flow, and check if the selected feature plugins were installed
- Go to **WooCommerce > Home** the task list should have **Set up WooCommerce Payments** and the **Choose payment methods** should not be displayed, although there should be an inbox card with **Set up additional payment providers** that allows you to set up additional providers.

Scenario # 2:
- Follow the flow in Scenario # 1, but select the CBD industry on the industry section
- When on the free business features tab, it should not be displaying the **WooCommerce Payments** option, and act similar as scenario # 3

Scenario # 3:
- Follow scenario # 1, but on the free features tab make sure that **WooCommerce Payments** is unselected.
- Finish the onboarding and go to the home screen
- It should display the **Choose payment methods** task, and include **WooCommerce Payments** as part of the list when clicking on this task
- The copy of WCPay should reflect the 3rd bullet in the acceptance criteria in https://github.com/woocommerce/woocommerce-admin/issues/6514
- If you select the CBD industry, it should not be displaying the **WooCommerce Payments** option as part of this list

Scenario # 4:
- Set the `wcpay/support-international-countries` feature flag to false in `config/development.json` (for local) and rebuild the app
- Follow Scenario # 1, and Woocommerce payment should not be in the list of free features

Scenario # 5:
- In a new store, or the current one WC Payments plugin disabled and uninstalled) go to the Onboarding flow (**Store details** task on home screen takes you there as well)
- Select one of the countries in this list: BR | FR | ID | DE | VN | PL | NG | GR | BE | PT | DK | SE | JP
- Select any one of the industries aside from the CBD option.
- Go the the business details tab, it should not be showing the checkbox at the bottom with the free features bundle
- Fill out the dropdown's and click continue
- It should show the `Included business features` tab, allowing you to expand the list, and unselect individual features
- Make sure that `WooCommerce Payments` is **NOT** part of this list, and should not be in the small print either (below the main card that tells you which plugins are being installed)
- Finish the onboarding flow, and check if the selected feature plugins were installed
- Go to **WooCommerce > Home** the task list should have **Choose payment methods** and the **Set up WooCommerce Payments** should not be displayed.

Scenario # 6:
- In a new store, or the current one WC Payments plugin disabled and uninstalled) go to the Onboarding flow (**Store details** task on home screen takes you there as well)
- Select one a country that wasn't in any of the previous lists (Spain is one for example)
- Select any one of the industries aside from the CBD option.
- It should not have the Free features tab, and instead show several features (4 I believe) under the dropdown questions that you can select/unselect.
- Fill out the dropdown's and click continue
- Finish the onboarding flow, and check if the selected feature plugins were installed
- Go to **WooCommerce > Home** the task list should have **Choose payment methods** and the **Set up WooCommerce Payments** should not be displayed.
